test/src/unit-cppapi-deletes.cc

```diff
@@ -246,6 +275,17 @@ void DeletesFx::write_sparse_v11(uint64_t timestamp) {
   array.close();
 }
 
+std::vector<tiledb::Object> DeletesFx::read_group(
+    const tiledb::Group& group) const {
+  std::vector<tiledb::Object> ret;
+  uint64_t count = group.member_count();
```

Comment:
NIT: ret.reserve(count);

This won't impact performance here but we should still add it to set the example :)